dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -896,6 +896,7 @@
           }
         },
         "confidenceScore": {
+          "x-ms-client-name": "confidence",
           "type": "number",
           "format": "double",
           "description": "Confidence score between 0 and 1 of the extracted relation."
```

Comment:
We should also say the format e..g know many decimal place (should be the same as sentiment)